tests/fixtures.py

```diff
@@ -10,13 +8,18 @@
 @fixture(name="dvc_repo_session", scope="session")
 def fix_dvc_repo_session() -> DvcRepo:
     with tempfile.TemporaryDirectory() as dir:
-        # os.mkdir("./temp_test")
         subprocess.check_call(["git", "init"], cwd=dir)
+        print(dir)
```

Comment:
if code is working good to remove. Also remove commented os.mkdir above.

Reply:
Woops. I'll go ahead and remove that